recipes/newrelic/apache/debian.yml

```diff
@@ -0,0 +1,117 @@
+name: apache-open-source-integration
+displayName: Apache Open Source Integration
+description: New Relic install recipe for default Apache Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-apache
+
+installTargets:
+  - type: host
+    os: linux
+    platform: "debian"
+  - type: host
+    os: linux
+    platform: "ubuntu"
+
+# keyword convention for dealing with search terms that could land someone on this instrumentation project
+keywords:
+  - Infrastructure
+  - Integration
+  - apache
+
+# Examine Infrastructure events for correlated data
+processMatch:
+  - apache2
+
+# Matches partial list of the Log forwarding parameters
+# https://docs.newrelic.com/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent#parameters
+logMatch:
+  - name: apache
+    file: /var/log/apache2/*access.log
+  - name: apache error
+    file: /var/log/apache2/error.log
+
+# NRQL the newrelic-cli will use to validate the agent/integration this recipe
+# installed is successfully sending data to New Relic
+validationNrql: "SELECT count(*) from ApacheSample where hostname like '{{.HOSTNAME}}' FACET entityGuid SINCE 10 minutes ago"
+
+# Prompts for input from the user. These variables then become
+# available to go-task in the form of {{.VAR_NAME}}
+inputVars:
+  - name: "NR_CLI_STATUS_URL"
+    prompt: "Please enter your apache status URL below. If none is provided then default value: http://127.0.0.1/server-status?auto will be used."
+    default: "http://127.0.0.1/server-status?auto"
+
+install:
+  version: "3"
+
+  silent: true
+
+  tasks:
+    default:
+      cmds:
+        - task: assert_pre_req
+        - task: setup
+        - task: restart
+        
+    assert_pre_req:
+      cmds:
+        - |
+          SERVICE_EXIST=$(sudo systemctl status newrelic-infra.service | grep "Active" | wc -l)
+          if [ $SERVICE_EXIST -eq 0 ]; then
+            echo "The newrelic-infra agent service is NOT installed on the host, but is required to install this integration." >> /dev/stderr
+            exit 1
+          fi
+
```

Comment:
@JakubKotkowiak similarly to nginx, there is a statusURL needed so the integration can work properly.
Could you add a pre-req that test the statusURL responds correctly, and if not echo an explicit error message and return a different exit code?